src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -442,6 +444,54 @@ private function addTemplatingSection(ArrayNodeDefinition $rootNode)
         ;
     }
 
+    private function addAssetsSection(ArrayNodeDefinition $rootNode)
+    {
```

Comment:
This code reminds me http://www.history.com/topics/ancient-history/the-egyptian-pyramids, :dancers: